libs/blocks/marketo/marketo.css

```diff
@@ -162,11 +163,63 @@
   font-weight: 400;
 }
 
+.marketo .mktoForm .mktoCheckboxList input {
+  appearance: none;
+  border: 0;
+  border-radius: 2px;
+  padding: 0;
+  width: 14px!important;
```

Comment:
I wasn't loving this `!important` so I had a quick look - you can skip declaring the `width` and `height` here altogether. Then, remove absolute positioning (and `top`/`left` declarations as well, of course) from `.marketo .mktoForm .mktoCheckboxList input::before`, which will then fit perfectly into its parent. It already has a width and height of `10px` + `2px` border on each side, which totals `14px`, exactly as this element. This leads to the similar UI, but avoids the `!important` declaration

Reply:
Thanks!